integrationExamples/gpt/geoedgeRtdProvider_example.html

```diff
@@ -0,0 +1,104 @@
+<!--
+  This page calls a single bidder for a single ad slot. It can be considered a "hello world" example for using
+  Prebid with the Google Publisher Tag.
+  It also makes a good test page for new adapter PR submissions. Simply set your server's Bid Params object in the
+  bids array inside the adUnits, and it will use your adapter to load an ad.
+  NOTE that many ad servers won't send back an ad if the URL is localhost... so you might need to
+  set an alias in your /etc/hosts file so that you can load this page from a different domain.
+-->
```

Comment:
Please change the opening comment for this example html page. Make it more suited to your module and also add inline comments.

Personally, I find the `geoedgeRtdProvider.md` descriptive enough which makes this html example file redundant. And your configuration looks simple enough to understand without looking at this example.

I would've preferred an example html page, where all current RTD modules' usage is demonstrated. instead of having a page for each RTD module. But, that'll be a different PR. @bretg Any inputs here?

Reply:
We changed the opening comment and added inline comments. Please let us know if you prefer to remove it entirely and leave only `geoedgeRtdProvider.md`, or submitting a new PR for a dedicated all RTD providers example page.